README.md

```diff
@@ -65,6 +65,15 @@ yarn storybook
 
 When developing a new component, always add a story for it. If you were creating a component like `src/components/StarIcon/index.tsx` then you'd put its story in `stories/StarIcon.stories.tsx`. The storybook server will automatically load files having the `.stories.tsx` suffix in this directory.
 
+## Configuration
+
+You can configure the app by defining environment variables in `.env` files, the standard way to [configure Create React App](https://facebook.github.io/create-react-app/docs/adding-custom-environment-variables#adding-development-environment-variables-in-env).
+
+- How to add a new environment variable
+  - Add it to `.env` with the `REACT_APP_` prefix.
+- How to override an environment variable for a hosted site
+  - Define the variable in the [corresponding puppet config](https://github.com/mozilla-services/cloudops-deployment/tree/master/projects/addons-code-manager/puppet/yaml/type). To define a variable for `code.addons-dev.allizom.org`, for example, you'd update `amo.code_manager.dev.yaml`. Adding a variable to `.env.dev` **will do nothing** since that only affects the `yarn dev` command.
```

Comment:
Should this be "[corresponding puppet config file]", as it is a file that needs to be edited, correct?